src/file.rs

```diff
@@ -69,6 +74,19 @@ impl FileLoggerBuilder {
         self
     }
 
+    /// Sets KVFilter
```

Comment:
For users that unfamiliar with this filter, I think that it is kind to add a link to the docs.rs page as follows.

```rust
/// Sets [`KVFilter`].
///
/// [`KVFilter`]: https://docs.rs/slog-kvfilter/0.6/slog_kvfilter/struct.KVFilter.html
```

Reply:
sure ,you want me to add it ?

Reply:
Yes, please. > you want me to add it ?

Reply:
Done